core/kazoo_stdlib/include/kz_types.hrl

```diff
@@ -118,10 +118,11 @@
 -type kz_ip_list() :: ne_binaries().
 
 %% Recreate the non-exported types defined in the erlang supervisor source
--type sup_child_spec() :: supervisor:child_spec().
+-type sup_child_spec() :: supervisor:child_spec() | list().
```

Comment:
Are you sure?

Reply:
Left-over from fixing start_child specs for sofo children...reverting